templates/example_fiber_segmentation.ipynb

```diff
@@ -0,0 +1,190 @@
+{
```

Comment:
<p>This should be updated for the new language we have about mounting external dirs etc. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>